Introduces instance level skip validation option. (#644)
aasm/aasm: README.md

end
```

Also You can skip the validation at instance level with `some_event_name_without_validation!` method.
With this you have the flexibility of having validation for all your transitions by default and then skip it wherever required.

> Please not that only update column will be updated as mentioned in the above example.

Reviewer: ```suggestion
Please note that only state column will be updated as mentioned in the above example.
```

<br/>

> <br/>

Reviewer: ```suggestion
```

> <br/>

Reviewer: ```suggestion
```
